molgenis-data-mapper/src/main/java/org/molgenis/data/mapper/algorithmgenerator/categorymapper/convertor/SeveralTimesConvertor.java

```diff
@@ -26,9 +26,9 @@ AmountWrapper getInternalAmount(String description)
 		Unit<?> unit = CategoryMapperUtil.findDurationUnit(description);
 		if (unit != null && unit.isCompatible(STANDARD_UNIT))
 		{
-			return AmountWrapper.create(
-					Amount.rangeOf((double) 2, NonSI.DAY.inverse().getConverterTo(unit).convert(1) - 1, unit).to(
-							STANDARD_UNIT), false);
+			return AmountWrapper
+					.create(Amount.rangeOf((double) 3, NonSI.DAY.inverse().getConverterTo(unit).convert(1) - 1, unit)
+							.to(STANDARD_UNIT), false);
```

Comment:
Please change the STANDARD_UNIT to a more logical name,  that reflects NonSI.WEEK.inverse() or use NonSI.WEEK.inverse().
Please add some comments to this algorithm. Why dit you choose/changed to minmum 3?